src/olympia/translations/models.py

```diff
@@ -209,21 +216,42 @@ def clean(self):
         self.localized_string_clean = clean_nl(cleaned).strip()
 
     def clean_localized_string(self):
-        # All links (text and markup) are normalized.
-        linkify_filter = partial(
-            bleach.linkifier.LinkifyFilter,
-            callbacks=[linkify_bounce_url_callback, bleach.callbacks.nofollow],
-        )
         # Keep only the allowed tags and attributes, escape the rest.
         cleaner = bleach.Cleaner(
             tags=self.allowed_tags,
             attributes=self.allowed_attributes,
-            filters=[linkify_filter],
+            filters=[self.linkify_filter],
         )
 
         return cleaner.clean(str(self.localized_string))
 
 
+class PurifiedMarkdownTranslation(PurifiedTranslation):
+    class Meta:
+        proxy = True
+
+    def clean_localized_string(self):
+        # bleach user-inputted html
+        cleaned = (
+            bleach.clean(self.localized_string, tags=[], attributes={})
+            if self.localized_string
+            else ''
+        )
+        # hack; cleaning breaks blockquotes
+        text_with_brs = cleaned.replace('&gt;', '>')
+        markdown = md.markdown(text_with_brs, extensions=['abbr', 'fenced_code'])
```

Comment:
It's not clear why we are passing an array with "abbr" and "fenced_code" here.. can you add a comment?